index.d.ts

```diff
@@ -8,7 +8,7 @@ declare module 'react' {
 }
 
 export type StyleRegistry = {
-  styles(options?: { nonce?: boolean }): JSX.Element[]
+  styles(options?: { nonce?: string | boolean }): JSX.Element[]
```

Comment:
What is the expected behavior when `nonce: true` is passed?

Reply:
`nonce="true"` boolean is still coverted to string, we can also change to string if we don't need the back compatible since it's bit trivial

Reply:
Should `nonce="true"` be rendered to the DOM, doesn't the nonce need to match the header/meta tag to be valid?